app/models/ability.rb

```diff
@@ -1,11 +1,14 @@
 class Ability
   include CanCan::Ability
+
   def initialize(user)
     user ||= User.new
     if user.admin?
       can :view, :all
       can :manage, Order
     else
+      can :manage, Business, user_id: user.id
+      can :create, Business
```

Comment:
If the admin is going to be able to change the `business.status` when it starts as pending, the admin would need to be able to `:manage` the businesses. This will also be required to give the admin the privilege to delete the business.